srtcore/srt.h

```diff
@@ -724,6 +783,7 @@ SRT_API int srt_bistats(SRTSOCKET u, SRT_TRACEBSTATS * perf, int clear, int inst
 SRT_API SRT_SOCKSTATUS srt_getsockstate(SRTSOCKET u);
 
 SRT_API int srt_epoll_create(void);
+SRT_API extern int srt_epoll_clear_usocks(int eid);
```

Comment:
Why is 'extern' needed here?

Reply:
Likely, it was there for every function before. Should be removed.